tests/Query/Filter/TagsField.test.ts

```diff
@@ -8,6 +8,34 @@ import { Sort } from '../../../src/Query/Sort';
 import { TaskBuilder } from '../../TestingTools/TaskBuilder';
 import { expectTaskComparesAfter, expectTaskComparesBefore } from '../../CustomMatchers/CustomMatchersForSorting';
 
+describe('tag presence & absence', () => {
+    it.each(['has tag', 'has tags'])('should have "%s" filtering', (filterLine: string) => {
+        // Arrange
+        const filter = new TagsField().createFilterOrErrorMessage(filterLine);
+
+        // Act, Assert
+        expect(filter.filterFunction).toBeDefined();
+        expect(filter.error).toBeUndefined();
+
+        expect(filter).toMatchTaskFromLine('- [ ] stuff #one');
+        expect(filter).toMatchTaskFromLine('- [ ] stuff #one #two');
+        expect(filter).not.toMatchTaskFromLine('- [ ] no tag here');
```

Comment:
Could there be at least one test that shows how this interacts with the global filter please?

Reply:
I don’t know off the top of my head. 

I saw today somewhere in the docs an explanation of how tags and global filter interact. So suggest looking for that.

Reply:
Maybe ‘no tags’ should mean ‘no tags other than the global filter’?

Reply:
it's ok, I figured it out. I've fallen victim to the fact that the GF can be only at the beginning of the task... I also found a way to fix that =D (but it will be for another PR)

Reply:
> Maybe ‘no tags’ should mean ‘no tags other than the global filter’?

correct